index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>UPB-1</title>
+
+    <link rel="stylesheet" href="./CSS/main.css">
+</head>
+<body>
+    <section class="main">
+        <header>
+            <h1 class="header__text">GAWSS</h1>
+        </header>
+        <section class="first__section">
+            <div class="bigBox">
+                <h2 class="box__text">Big Box 1</h2>
+            </div>
+            <div class="bigBox">
+                <h2 class="box__text">Big Box 2</h2>
+            </div>
+        </section>
+        <section class="second__section">
+            <div class="smallBox">
+                <h3 class="box__text">Small Box 1</h2>
+            </div>
+            <div class="smallBox">
+                <h3 class="box__text">Small Box 2</h2>
+            </div>
+            <div class="smallBox">
+                <h3 class="box__text">Small Box 3</h2>
+            </div>
+        </section>
+        <footer class="footer">
+            <div class="bottomBox"></div>
```

Comment:
If the parent of this div it's called "footer" and you should use BEM, then this selector should be called differently.